WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -586,7 +586,7 @@ private void deserializeJetpackRestResponse(SiteModel site, JSONObject response)
             params.put("protect", mJpSettings.jetpackProtectEnabled);
         }
         if (!mJpSettings.whitelistMatches(mRemoteJpSettings.jetpackProtectWhitelist)) {
-            JSONArray whitelistArray = new JSONArray(mJpSettings.jetpackProtectWhitelist);
+            String whitelistArray = mJpSettings.jetpackProtectWhitelist.toString().replace("[", "").replace("]", "");
```

Comment:
Is there a reason we are not using this: `TextUtils.join(", ", mJpSettings.jetpackProtectWhitelist);`?